File: fe2-android/app/src/test/unit/java/com/github/dedis/popstellar/model/network/method/UnsubscribeTest.java
package com.github.dedis.popstellar.model.network.method;

import com.github.dedis.popstellar.model.objects.Channel;

import org.junit.Test;

import static org.junit.Assert.assertEquals;

public class UnsubscribeTest {

  @Test
  public void getMethod() {
    Unsubscribe unsubscribe = new Unsubscribe(Channel.fromString("root/stuff"), 3);
    assertEquals("unsubscribe", unsubscribe.getMethod());
  }
}

File: fe2-android/app/src/test/unit/java/com/github/dedis/popstellar/model/network/method/UnsubscribeTest.kt
package com.github.dedis.popstellar.model.network.method

import com.github.dedis.popstellar.model.objects.Channel.Companion.fromString
import org.junit.Assert
import org.junit.Test

class UnsubscribeTest {
  @Test
  fun method() {
    val unsubscribe = Unsubscribe(fromString("root/stuff"), 3)
    Assert.assertEquals("unsubscribe", unsubscribe.method)
  }
}
